config/jobs/etcd-druid/druid-e2e-kind.yaml

```
@@ -50,7 +50,7 @@ periodics:
       testgrid-days-of-results: "60"
     spec:
       containers:
-        - image: gcr.io/k8s-staging-test-infra/v20230421-ec4335b54b-master
+        - image: gcr.io/k8s-staging-test-infra/krte:v20230421-ec4335b54b-master
```

Can you change it latest?
```suggestion
        - image: gcr.io/k8s-staging-test-infra/krte:v20230613-63d85f5ed2-master
```